addons/sourcemod/scripting/vsh/bosses/boss_seeldier.sp

```diff
@@ -31,16 +31,16 @@ methodmap CSeeldier < SaxtonHaleBase
 	
 	public void GetBossInfo(char[] sInfo, int length)
 	{
-		StrCat(sInfo, length, "\nDuo Boss with SeeMan");
+		StrCat(sInfo, length, "\nDuo Boss with Seeman");
 		StrCat(sInfo, length, "\nMelee deals 124 damage");
 		StrCat(sInfo, length, "\nHealth: Low");
 		StrCat(sInfo, length, "\n ");
 		StrCat(sInfo, length, "\nAbilities");
 		StrCat(sInfo, length, "\n- Brave Jump");
 		StrCat(sInfo, length, "\n ");
 		StrCat(sInfo, length, "\nRage");
-		StrCat(sInfo, length, "\n- Summons 3 mini seeldiers");
-		StrCat(sInfo, length, "\n- 200%% Rage: Summons 6 mini seeldiers");
+		StrCat(sInfo, length, "\n- Summons 3 mini-SSeeldiers");
```

Comment:
Double S

Reply:
why the double S? Thought it didn't have that

Reply:
nevermind i am a moron and is fixed now